docarray/array/storage/elastic/seqlike.py

```diff
@@ -58,34 +58,47 @@ def __repr__(self):
         """
         return f'<{self.__class__.__name__} (length={len(self)}) at {id(self)}>'
 
+    def _parse_failed_index_ids_from_bulk_info(
```

Comment:
```suggestion
    def _parse_index_ids_from_bulk_info(
```

Could you rename it to _parse_index_ids_from_bulk_info ? The function is more generic than failed index. And could you make it a staticmethod ?

Reply:
done pls rereview